ext/json/ext/vendor/fpconv/README.md

```diff
@@ -0,0 +1,3 @@
+The contents of this directory is extracted from https://github.com/night-shift/fpconv
+
+It is licensed under the provisions of the Boost Software License - Version 1.0 - August 17th, 2003. See the ./license file for details.
```

Comment:
Maybe it's simpler to just add the license as comments at the top of `fpconv.c`?

Reply:
This license applies to three files, not just one. Do you think I should still move this into those files?